packages/cli/src/cmds/validator/slashingProtection/export.ts

```diff
@@ -51,7 +52,6 @@ export const exportCmd: CliCommand<ExportArgs, ISlashingProtectionArgs & Account
 
     handler: async (args) => {
       const {file} = args;
-      if (!file) throw new YargsError("must provide file arg");
```

Comment:
`demandOption: true` already ensures that a file arg is provided

https://github.com/ChainSafe/lodestar/blob/227930cdf61d266d25df466ea08a53a955114a5b/packages/cli/src/cmds/validator/slashingProtection/export.ts#L36